.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+# Use a package of configuration called an orb.
+orbs:
+  # Declare a dependency on the welcome-orb
+  welcome: circleci/welcome-orb@0.4.1
+# Orchestrate or schedule a set of jobs
+workflows:
+  # Name the workflow "welcome"
+  welcome:
+    # Run the welcome/run job in its own container
+    jobs:
+      - welcome/run
```

Comment:
I'm not sure this was meant to be part of this change. Would you mind removing it?

Reply:
Without this file, CircleCI [build ](ci/circleci: Build Error — Your tests failed on CircleCI)
 failed as soon as i pushed my code.  Have removed the file this time.

Reply:
It looks like CircleCI was using a cached configuration from the original Lansing Codes API to try to run tests in this project. Since this new API isn't using CircleCI for testing, I canceled that build so hopefully it won't continue to run and fail. 🤞